### PR TITLE
New version

### PR DESCRIPTION
**Pull request by matiasmarrapodi**

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 4.0"
```
Comment (ivours) [inline versions.tf]: Terraform v4 doesn't exist. Do we need to update the Terraform version constraint?

File: versions.tf [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.6.0"
+      version = ">= 4.60.0"
```
Comment (ivours) [inline versions.tf]: Please use a constraint that keeps the same major AWS provider version, like `version = "~> 4.0"`

File: versions.tf (hunk 1 above)
Comment (matiasmarrapodi) [reply versions.tf]: I remove that change

File: versions.tf (hunk 2 above)
Comment (matiasmarrapodi) [reply versions.tf]: Fixed

**Review by ivours — APPROVED** [review]

**Pull request merged**